mysql-test/suite/innodb/t/percona_changed_page_bmp_1364315.test

```diff
@@ -0,0 +1,38 @@
+#
+# Test that I_S.INNODB_CHANGED_PAGES works if a trailing path separator
+# omitted in innodb_data_home_dir (bug 1364315)
+
+--source include/have_innodb.inc
+
+# Delete any existing bitmaps
+--source include/restart_mysqld.inc
```

Comment:
Delete bitmaps != server restart, in any case here we don't need to do neither. Existing bitmaps would be benign for this testcase